src/settings-ui/Settings.UI/SettingsXAML/Views/DashboardPage.xaml

```diff
@@ -281,10 +281,10 @@
             VerticalAlignment="Center"
             CornerRadius="8"
             IsClosable="False"
-            IsOpen="{x:Bind ViewModel.UpdateAvailable, Mode=OneWay}"
+            IsOpen="True"
```

Comment:
I assume `True` is used for testing purpose?

Reply:
Nice catch. Thought I have reverted that.